testsuite/features/secondary/srv_menu_filter.feature

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@scope_visualization
+Feature: Left Menu Filter
+  In order to verify the left menu filter works properly
+  As an authorized user
+  I want to be able to search different keywords in the menu filter 
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+  
+  Scenario: Search a word and expect the right result
+    When I enter "Admin" as the left menu search field
+    Then I should see a "Admin" link in the left menu  
+
+  Scenario: Search a partial word and expect the right result
+    When I enter "Formula" as the left menu search field
+    Then I should see a "Formula Catalog" link in the left menu
+
+  Scenario: Search a word with a different case and expect the right result
+    When I enter "saLT" as the left menu search field
+    Then I should see a "Salt" link in the left menu  
+
+@susemanager
```

Comment:
I only now notice that, I would say this is bad indented, I think need to be at same indentation that the scenario title.
But I'm not totally sure. In any case, fine to merge this port for now. If it did not failed in 4.2 CI

Reply:
Let's fix it here @elariekerboull , and then we port this to other branches.
Sorry to catch this a bit late.

Reply:
I had another look at our code and it seems we have a mixed indentation regarding the tags.